easybuild/easyconfigs/c/Chemaxon-Marvin/Chemaxon-Marvin-23.9.eb

```diff
@@ -0,0 +1,72 @@
+##
+# This is a contribution from SIB Swiss Institute of Bioinformatics
+# Homepage:     https://www.sib.swiss/research-infrastructure/competence-centers/vital-it
+#
+# Authors::     Sebastien Moretti <sebastien.moretti@sib.swiss>
+#
+##
+
+# Cannot use *easyblock = 'Rpm'* because the RPM misses some fields:
+# Summary and License fields must be present in package: (main package)
+easyblock = 'Binary'
+
+name = 'Chemaxon-Marvin'
+version = '23.9'
+
+homepage = 'https://chemaxon.com/products/marvin'
+description = """Marvin suite is a chemically intelligent desktop toolkit built
+to help you draw, edit, publish, render, import and export your chemical structures
+and as well as allowing you to convert between various chemical and graphical file
+formats. It is free for individual, academic and non-commercial use."""
+
+# NOTE not in the license list of EB
+# software_license = 'Proprietary: Personal, Commercial or Academic'
+software_license_urls = ['https://www.chemaxon.com/marvin/license.html']
+docurls = [
+    'https://docs.chemaxon.com/display/docs/marvinsketch-user-s-guide.md',
+    'https://docs.chemaxon.com/display/docs/marvinview-user-s-guide.md',
+    'https://docs.chemaxon.com/display/docs/file-formats.md',
+]
+
+toolchain = SYSTEM
+
+# Download and license at https://chemaxon.com/products/marvin
```

Comment:
Not clear that I can use any of their license options for testing, really.

Reply:
Maybe better now with updated URLs.

But this tool is like matlab, you may not have access to the source package. That's why for the first eb file of this tool they asked me to upload a test-report. What I did this morning

Reply:
Sure. If we can only provide limited testing, that's no reason not to include it.